scratch/PCA/extract_data.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
If others are not expected to pay attention to the `*.ipynb`, then I'd suggest not adding them (in which case having `*.ipynb` in the `.gitignore` makes sense).  Deleting the file may be a better option than continuing to track changes since they'll add noise to the PRs.

(This isn't a big deal though, so don't let it block you; I'm more just sticking my nose in to learn more about your workflow).